### PR TITLE
fix: add network tags to auto provision nodes

### PR DESCRIPTION
**Pull request by rochana-atapattu**
Adding the network tag used by the firewall rule to the auto provisioned nodes #1230

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@rochana-atapattu
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@rochana-atapattu
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: autogen/main/cluster.tf.tmpl [hunk 1]
```diff
@@ -469,6 +469,13 @@ resource "google_container_cluster" "primary" {
       topic   = var.notification_config_topic
     }
   }
+  {% if beta_cluster %}
+  node_pool_auto_config {
```
Comment (bharathkkb) [inline autogen/main/cluster.tf.tmpl:473]: Could we make this a dynamic block enabled only if var.cluster_autoscaling.enabled for beta clusters?

**Review by bharathkkb — COMMENTED** [review]
Thanks for the PR @rochana-atapattu

**Pull request closed without merging**